server/master/src/main/java/org/apache/accumulo/master/tserverOps/ShutdownTServer.java

```diff
@@ -52,12 +52,7 @@ public long isReady(long tid, Master master) throws Exception {
     }
 
     // Inform the master that we want this server to shutdown
-    // We don't want to spam the master with shutdown requests, so
-    // only send this request once
-    if (!requestedShutdown) {
-      master.shutdownTServer(server);
-      requestedShutdown = true;
```

Comment:
This state is never actually persisted as a new one of this object is created between each isReady call.